test/blackbox-tests/test-cases/actions/cond.t

```diff
@@ -0,0 +1,134 @@
+Testing (case) action.
+
+  $ cat > dune-project << EOF
+  > (lang dune 3.11)
+  > EOF
+
+Basic usage:
+
+  $ cat > dune << EOF
+  > (rule
+  >  (alias foo)
+  >  (action
+  >   (cond
+  >    ((= 1 2) (echo " 1 is 2"))
+  >    ((= 2 3) (echo "2 is 3"))
+  >    ((< 3 4) (echo "3 is less than 4"))
+  >    (_ (echo "shouldn't happen")))))
+  > EOF
+
+  $ dune build @foo
+  3 is less than 4
+
+Missing default field:
+
+  $ cat > dune << EOF
+  > (rule
+  >  (alias foo)
+  >  (action
+  >   (cond
+  >    ((= 1 2) (echo " 1 is 2")))))
+  > EOF
+
+  $ dune build @foo
+  File "dune", line 5, characters 4-11:
+  5 |    ((= 1 2) (echo " 1 is 2")))))
+          ^^^^^^^
+  Error: Only the default case can be at the end.
```

Comment:
This error message would be clearer if it read `Case statements must end with a default case.` to clarify that the real problem is the lack of default case - not putting a non-default case at the end.

Reply:
Also since there is another similar feature which uses the keyword `case` we should avoid using the word "case" in error messages relating to the `cond` feature to avoid confusion between the two features.

Reply:
I've fixed both these issues and opted to use the word "branch".